packages/react/src/themes/teams/components/RadioGroup/radioGroupItemVariables.ts

```diff
@@ -1,29 +1,41 @@
 import { pxToRem } from '../../../../lib'
 
 export type RadioGroupItemVariables = {
-  color: string
-  colorChecked: string
   colorDisabled: string
 
-  colorBorder: string
-  colorBorderChecked: string
+  // can these be global colors so we don't have to define for every component?
```

Comment:
Any suggestions? Almost every component will need `focusInnerBorderColor` and `focusOuterBorderColor`, is there a better way to define so it's not added to all the variables?

Reply:
After the color scheme this will be aligned with the color scheme used per theme. Let's restrain for making some changes at this moment.

Reply:
Btw, it is interesting that there are two border colors (maybe we will have to define border and outline colors.